API/controller/orderController.js

```diff
@@ -0,0 +1,23 @@
+import dummydb from '../Dummydb/dummydb';
+
+
+// TO GET ALL ORDERS
+const getAllOrders = (req, res) => {
+  if (!dummydb.orders) {
+    res.json({
```

Comment:
Add a return keyword here. That prevents the remainder of the code from running if this if statement is evaluated